### PR TITLE
dm: update doc for error handling

### PR DESCRIPTION
**Pull request by ran-huang**
Signed-off-by: Ran <huangran@pingcap.com>

### What is changed, added or deleted? (Required)

DM adds or updates commands about binlog, binlog-schema, shard-ddl-lock, config. They will replace handle-error, operate-schema, unlock-ddl-lock, show-ddl-locks, get-config to make commands more accessible.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/8683, https://github.com/pingcap/docs-cn/pull/8715
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: dm/handle-failed-ddl-statements.md [hunk 1]
```diff
@@ -8,77 +8,111 @@ aliases: ['/docs/tidb-data-migration/dev/skip-or-replace-abnormal-sql-statements
...
 
 ### query-status
 
 The `query-status` command is used to query the current status of items such as the subtask and the relay unit in each MySQL instance. For details, see [query status](/dm/dm-query-status.md).
 
-### handle-error
+### binlog
 
-The `handle-error` command is used to handle the failed DDL statements.
+The `binlog` command is used to manage and show binlog operations. This command is only supported in DM v6.0 and later versions. For earlier versions, use the `handle-error` command.
```
Comment (WizardXiao) [inline dm/handle-failed-ddl-statements.md:41]:
```suggestion
The `binlog` command is used to manage and show binlog operations. This command is only supported in DM v6.0 and later versions. For earlier versions, the `handle-error` command can be used.
```

**Review by WizardXiao — APPROVED** [review]
LGTM

File: dm/dm-manage-schema.md [hunk 2]
```diff
@@ -30,76 +30,83 @@ However, during the migration with [optimistic mode sharding DDL support](/dm/fe
...
+manage or show table schema in schema tracker
 
 Usage:
-  dmctl operate-schema <operate-type> <-s source ...> <task-name | task-file> <-d database> <-t table> [schema-file] [--flush] [--sync] [flags]
+  dmctl binlog-schema [command]
+
+Available Commands:
+  delete      delete table schema structure
+  list        show table schema structure
+  update      update tables schema structure
```
Comment (hfxsd) [inline dm/dm-manage-schema.md:56]:
```suggestion
  update      update table schema structure
```

Comment (hfxsd) [inline dm/dm-manage-schema.md:56]: 和 L74 - L76 保持一致
```suggestion
  delete      deletes the table schema
  list        lists the table schema
  update      updates the table schema
```

File: dm/dm-manage-schema.md [hunk 3]
```diff
@@ -117,7 +124,32 @@ operate-schema get -s mysql-replica-01 task_single -d db_single -t t1
...
+To update the table schema, run the `binlog-schema update` command:
+
+{{< copyable "shell-regular" >}}
+
+```bash
+help binlog-schema update
+```
+
+```
+update tables schema structure
```
Comment (hfxsd) [inline dm/dm-manage-schema.md:138]:
```suggestion
update the table schema
```

File: dm/dm-manage-schema.md (hunk 2 above)
Comment (ran-huang) [reply dm/dm-manage-schema.md:56]: 这个命令的 help 输出就是这样的，想改的话得在代码仓库里改😂 
cc @WizardXiao

File: dm/dmctl-introduction.md [hunk 4]
```diff
@@ -41,26 +41,29 @@ Usage:
...
-  resume-relay    Resumes DM-worker's relay unit.
-  resume-task     Resumes a specified paused task.
-  show-ddl-locks  Shows un-resolved DDL locks.
-  start-task      Starts a task as defined in the configuration file.
-  stop-task       Stops a specified task.
-  unlock-ddl-lock Unlocks DDL lock forcefully.
+  binlog          manage or show binlog operations
+  binlog-schema   manage or show table schema in schema tracker
+  check-task      Checks the configuration file of the task
+  config          manage config operations
```
Comment (hfxsd) [inline dm/dmctl-introduction.md:47]:
```suggestion
  binlog          Manages or shows binlog operations
  binlog-schema   Manages or shows the table schema in schema tracker
  check-task      Checks the configuration file of the task
  config          Manages config operations
```

File: dm/dmctl-introduction.md [hunk 5]
```diff
@@ -91,36 +94,40 @@ export DM_MASTER_ADDR="172.16.30.14:8261"
...
-  --config Path to configuration file.
-  --master-addr Master API server addr.
-  --rpc-timeout RPC timeout, default is 10m.
-  --ssl-ca Path of file that contains list of trusted SSL CAs for connection.
-  --ssl-cert Path of file that contains X509 certificate in PEM format for connection.
-  --ssl-key Path of file that contains X509 key in PEM format for connection.
+  binlog          manage or show binlog operations
+  binlog-schema   manage or show table schema in schema tracker
+  check-task      Checks the configuration file of the task
+  config          manage config operations
```
Comment (hfxsd) [inline dm/dmctl-introduction.md:100]:
```suggestion
  binlog          Manages or shows binlog operations
  binlog-schema   Manages or shows the table schema in schema tracker
  check-task      Checks the configuration file of the task
  config          Manages config operations
```

File: dm/dmctl-introduction.md [hunk 6]
```diff
@@ -91,36 +94,40 @@ export DM_MASTER_ADDR="172.16.30.14:8261"
...
+  decrypt         Decrypts cipher text to plain text
+  encrypt         Encrypts plain text to cipher text
+  help            Gets help about any command
+  list-member     Lists member information
+  offline-member  Offlines member which has been closed
+  operate-leader  `evict`/`cancel-evict` the leader
+  operate-source  `create`/`update`/`stop`/`show` upstream MySQL/MariaDB source
+  pause-relay     Pauses DM-worker's relay unit
+  pause-task      Pauses a specified running task or all (sub)tasks bound to a source
```
Comment (hfxsd) [inline dm/dmctl-introduction.md:109]:
```suggestion
  pause-task      Pauses a specified running task or all tasks or subtasks bound to a source
```

File: dm/dmctl-introduction.md [hunk 7]
```diff
@@ -91,36 +94,40 @@ export DM_MASTER_ADDR="172.16.30.14:8261"
...
+  purge-relay     Purges relay log files of the DM-worker according to the specified filename
+  query-status    Queries task status
+  resume-relay    Resumes DM-worker's relay unit
+  resume-task     Resumes a specified paused task or all (sub)tasks bound to a source
```
Comment (hfxsd) [inline dm/dmctl-introduction.md:113]:
```suggestion
  resume-task     Resumes a specified paused task or all tasks or subtasks bound to a source
```

File: dm/dmctl-introduction.md [hunk 8]
```diff
@@ -91,36 +94,40 @@ export DM_MASTER_ADDR="172.16.30.14:8261"
...
+  shard-ddl-lock  maintain or show shard-ddl locks information
+  start-relay     Starts workers pulling relay log for a source
+  start-task      Starts a task as defined in the configuration file
+  stop-relay      Stops workers pulling relay log for a source
+  stop-task       Stops a specified task or all (sub)tasks bound to a source
```
Comment (hfxsd) [inline dm/dmctl-introduction.md:118]:
```suggestion
  stop-task       Stops a specified task or all tasks or subtasks bound to a source
```

File: dm/handle-failed-ddl-statements.md [hunk 9]
```diff
@@ -8,77 +8,111 @@ aliases: ['/docs/tidb-data-migration/dev/skip-or-replace-abnormal-sql-statements
...
+Currently, TiDB is not completely compatible with all MySQL syntax (see [the DDL statements supported by TiDB](/mysql-compatibility.md#ddl)). Therefore, when DM is migrating data from MySQL to TiDB and TiDB does not support the corresponding DDL statement, an error might occur and break the migration process. In this case, you can use the `binlog` command of DM to resume the migration.
 
 ## Restrictions
 
-If it is unacceptable in the actual production environment that the failed DDL statement is skipped in the downstream TiDB and it cannot be replaced with other DDL statements, then do not use this command.
+Do not use this command in the following situations:
+
+* It is unacceptable in the actual production environment that the failed DDL statement is skipped in the downstream TiDB.
+* The failed DDL statement cannot be replaced with other DDL statements.
+* Other DDL statements must not be injected into the downstream TiDB.
```
Comment (hfxsd) [inline dm/handle-failed-ddl-statements.md:19]: 这段话比较费解，双重否定比较多，读者被绕晕了。建议按照中文重新翻译。

File: migrate-with-more-columns-downstream.md [hunk 10]
```diff
@@ -66,30 +66,32 @@ In such cases, you can use the `operate-schema` command to set a table schema fo
     )
     ```
 
-2. Use the `operate-schema` command to set the table schema for the table to be migrated from the data source (At this time, the data migration task should be in the Paused state due to the above `Column count doesn't match` error).
+2. Use the `binlog-schema` command to set the table schema for the table to be migrated from the data source (At this time, the data migration task should be in the Paused state due to the above `Column count doesn't match` error).
```
Comment (hfxsd) [inline migrate-with-more-columns-downstream.md:69]:
```suggestion
2. Use the `binlog-schema` command to set the table schema for the table to be migrated from the data source. At this time, the data migration task should be in the Paused state due to the above `Column count doesn't match` error.
```

File: dm/dm-manage-schema.md (hunk 2 above)
Comment (WizardXiao) [reply dm/dm-manage-schema.md:56]: 嗯，是的，可以先不改命令的 help 输出的结果，L74 - L76 对命令和参数的解释单独表达也行，大部分解释与 help 的输出可能不一致，只是这里比较像吧 😂

File: dm/handle-failed-ddl-statements.md (hunk 9 above)
Comment (ran-huang) [inline dm/handle-failed-ddl-statements.md:19]:
```suggestion
If it is unacceptable in the actual production environment that the failed DDL statement is skipped in the downstream TiDB, if the DDL statement cannot be replaced with other DDL statements, and if it is unacceptable to inject other DDL statements, do not use this command.
```

Comment (ran-huang) [reply dm/handle-failed-ddl-statements.md:19]: @hfxsd PTAL

**Review by hfxsd — APPROVED** [review]
LGTM

**Pull request merged**